src/crystal/dwarf/strings.cr

```diff
@@ -13,8 +13,8 @@ module Crystal
       def decode(strp)
         # See if we can read it from the buffer
         if strp < @buffer.size
-          index = @buffer.index('\0'.ord, offset: strp)
-          return String.new(@buffer[strp, index - strp]) if index
+          index = @buffer.index('\0'.ord, offset: strp.to_i!)
+          return String.new(@buffer[strp.to_i!, index - strp.to_i!]) if index
```

Comment:
Why the unsafe conversion is used in this operations?

Is it because `strp` is guarented to be a small value and will never overflow in 32 bits? Since this is an argument of of a public method I think is better to use the overflowing operator.

Reply:
Same, because I did the quickest thing to make it work, I didn't put much thought into this.